crates/sui-indexer-alt-schema/migrations/2025-02-05-232254_coin_balance_buckets_object_id_desc/down.sql

```diff
@@ -0,0 +1,4 @@
+DROP INDEX IF EXISTS coin_balances_buckets_owner_type;
+
+CREATE INDEX IF NOT EXISTS coin_balances_buckets_owner_type
```

Comment:
Name the new and old indices differently to each other, otherwise these migrations will always delete whatever index exists and recreate it (even if it had the correct structure). We want to be able to run the steps in this migration manually on the production instance (once we have the all clear), and then have this script detect that those steps have run.